package.json

```diff
@@ -7,23 +8,53 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "predeploy": "npm run build-storybook",
+    "deploy-storybook": "gh-pages -d storybook-static",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "cypress:open": "cypress open",
+    "cypress:run:component": "cypress run --component",
+    "percy": "percy exec -- cypress run --component --config-file cypress.config.ts"
```

Comment:
bug/suggestion: I can't run percy locally because it's missing a token. I recommend you to add a guide about testing/percy on the README (and if possible share a token with me on discord just for testing purposes). The bonus would be to connect percy to CI ;)

Reply:
added percy to CI 😎